desktop-src/updateorchestrator/index.md

```diff
@@ -36,3 +36,46 @@ The initial release of UpdateOrchestrator API as limited access feature is only
 
 Our aim is to improve the functionality of this API and reduce impact from multiple automatic software updaters on Windows. We would appreciate your input through this [**brief survey**](https://aka.ms/UOAPISurvey) to help us understand how UpdateOrchestrator API can better serve your developer needs.
 
+## For OEMs wanting to Expedite Applications 
+
+>[!IMPORTANT]
+> Expediting Applications is only available for OEMs and only applicable in select regions. Expediting an application may have negative impacts to the out of box experience for new devices. This functionality is only available on select client builds.
+
+OEMs can register an update to be expedited by writing a new key in the existing staging folder for expedited apps: HKLM\Software\Microsoft\WindowsUpdate\Orchestrator\UScheduler_OOBE 
```

Comment:
We also need to add the prerequisities for the apps that can be expedited in this manner:

To plug into this expedited flow, the desired app must meet the following requirements:
1. It must be a Store packaged app in MSIX format
2. It must have a valid Product Family Name (PFN)